modules/apps/portal-crypto-hash/portal-crypto-hash-api/configs/com.liferay.portal.crypto.hash.configuration.CryptoHashProcessorConfiguration-default.config

```diff
@@ -0,0 +1,2 @@
+configuration.name="Default Crypto Hash Processor Configuration"
```

Comment:
It looks this pull is introducing a bundled configuration file
here.


If that is the case and this pull is fixing a bug please mark this pull as 
*DO NOT BACKPORT*


Please see https://issues.liferay.com/browse/PTR-1770